File: Dev/C++/Mascaret/src/VEHA/Entity/ShapeProperty.cpp
#include "VEHA/Entity/ShapeProperty.h"
#include "MascaretApplication.h"
#include "VEHA/Kernel/Model.h"
#include "VEHA/Kernel/PrimitiveType.h"
namespace VEHA
{

ShapeProperty::ShapeProperty( const string& name,Class* cl)
:Property(name,cl,MascaretApplication::getInstance()->getModel()->getBasicType("shape"))
{
}

ShapeProperty::~ShapeProperty()
{
}

}

File: Dev/CS/Mascaret/Mascaret/VEHA/Entity/ShapeProperty.cs

using System;
using System.Collections.Generic;

namespace Mascaret
{
    public class ShapeProperty : Property
    {
        public ShapeProperty(string name, Class classe)
            : base(name, classe, MascaretApplication.Instance.Model.getBasicType("shape"), null, null, null)
        {
        }

    }
}


